>Stress test (thread) — Killed by signal (in clickhouse-server.log)

#21305 (`ZstdDeflatingWriteBuffer`)

>Integration tests (release) — fail: 0, passed: 460, error: 26

```
E           ERROR: for hdfs1  Cannot start service hdfs1: driver failed programming external connectivity on endpoint rootteststoragehdfs_hdfs1_1 (c19f177bf04279dc920c4d5cfea6da1ea124a104e1f0f6eff282c16176df690e): Error starting userland proxy: listen tcp4 0.0.0.0:50070: bind: address already in use
```

Should be addressed by #20393
Does not looks related anyway

--- a/src/Databases/DatabaseReplicatedWorker.cpp
+++ b/src/Databases/DatabaseReplicatedWorker.cpp
@@ -63,7 +63,7 @@ void DatabaseReplicatedDDLWorker::initializeReplication()
     if (our_log_ptr == 0 || our_log_ptr + logs_to_keep < max_log_ptr)
         database->recoverLostReplica(current_zookeeper, our_log_ptr, max_log_ptr);
     else
-        last_skipped_entry_name.emplace(log_ptr_str);
+        last_skipped_entry_name.emplace(DDLTaskBase::getLogEntryName(our_log_ptr));
 }
 
 String DatabaseReplicatedDDLWorker::enqueueQuery(DDLLogEntry & entry)

